libeos-payg/provider.c

```diff
@@ -154,6 +154,24 @@ epg_provider_default_init (EpgProviderInterface *iface)
                            G_PARAM_READABLE | G_PARAM_STATIC_STRINGS);
   g_object_interface_install_property (iface, pspec);
 
+  /**
+   * EpgProvider:code-length:
+   *
+   * The length of an unlock code.
```

Comment:
```suggestion
   * The length of an unlock code, in Unicode characters.
```

Should probably be explicit about whether the length is in terms of characters, or bytes. Characters would be more flexible for accepting emojis in future (for example).